src/assets/images/landing/README.md

```diff
@@ -1,6 +1,7 @@
 # Image Credits
 
-IMG_0894.JPG - Marina D. A. Scarpelli
-IMG_1316.JPG - Marina D. A. Scarpelli
-IMG_1318.JPG - Marina D. A. Scarpelli
-IMG_20180710_081824508_HDR.jpg - David Tucker
+IMG_dirt.JPG - Marina D. A. Scarpelli
+IMG_night.JPG - Marina D. A. Scarpelli
+IMG_road.JPG - Marina D. A. Scarpelli
+IMG_sky.JPG - David Tucker
+IMG_sunset.JPG - David Tucker
```

Comment:
encode this somewhere so the correct credit is shown for fallback images

Reply:
Tracking here: https://github.com/QutEcoacoustics/workbench-client/issues/1225